### DISCUSSION TITLE
AntBlazor Chart

### DISCUSSION THREAD

**Discussion by Wavesolid** [ant-design-blazor/ant-design-blazor#3332] [Q&A] [answered]
I tried the Gauge Chart from Ant Blazor, but it seems the docs is obsolete and not relevant with the exisisting component. Is there any idea to assign value to Gauge Chart?

**Comment by ElderJames** [comment, marked as answer]
Please take a look at https://github.com/ant-design-blazor/ant-design-charts-blazor/blob/49a967a6ecd9785b7b61deb052cb2be1a4e68f29/src/AntDesign.Charts.Docs/Demos/Gauge/Gauge_Chart.razor

**Reply by Wavesolid** [reply]
thank you for the reply, but it seems still not working on my Blazor WASM. The Gauge chart won't show up and when I opened up the console, it returns me an error like this
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/58335667/87831bc7-9e18-4812-b218-d3fca0a6a5d4).

Here's my code:
```
@page "/"

<PageTitle>Index</PageTitle>

<Gauge Config="_config2" />

@code {
private readonly GaugeConfig _config2 = new()
{
    Percent = new decimal(0.75),
    Radius = 0.75,
    Range = new GaugeRange
    {
        Color = "#30BF78",
        Width = 12,
    },
    Indicator = new GaugeIndicator()
    {
        Pointer = new GaugeIndicatorStyle
        {
            Style = new GaugeStyle
            {
                Stroke = "#D0D0D0"
            }
        },
        Pin = new GaugeIndicatorStyle
        {
            Style = new GaugeStyle
            {
                Stroke = "#D0D0D0"
            }
        }
    },
    Statistic = new GaugeStatistic()
    {
        Content = new GaugeStatisticStyle()
        {
            Content = "Rate: 75%",
        }
    },
    GaugeStyle = new GaugeStyle()
    {
        LineCap = "round"
    }
};
}
```

**Reply by ElderJames** [reply]
Which version of G2Plot are you using? Please use the 2.x version.

**Reply by Wavesolid** [reply]
It's working now, I forgot to change the G2Plot to the latest, according to the link that u referred. Previously I used the version according from the [docs](https://antblazor.com/en-US/charts/introduce). Thank you for your assist.
